modules/encrypt/classes/Kohana/Encrypt/Engine/Mcrypt.php

```diff
@@ -176,42 +135,10 @@ public function create_iv(): string
     {
         // Create a random initialization vector of the proper size for the current cipher
         $iv = mcrypt_create_iv($this->_iv_size, Encrypt_Engine_Mcrypt::$_rand);
-        if ($iv)
-        {
-            return $iv;
-        }
-
-        throw new Kohana_Exception('Could not generate random iv');
-    }
-
-    /**
-     * Normalize key for PHP 5.6 for backwards compatibility
-     *
-     * This method is a shim to make PHP 5.6 behave in a B/C way for
-     * legacy key padding when shorter-than-supported keys are used
-     *
-     * @param   string $key encryption key
-     * @param   string $cipher mcrypt cipher
-     * @param   string $mode mcrypt mode
-     * @return bool|string
-     */
-    protected function _normalize_key($key, $cipher, $mode)
```

Comment:
Where did this function go? I don't want to modify anything in this engine, in case something breaks backwards compatibility. Let's just leave this as it is, ok?

Reply:
I can but this function was only needed for php <= 5.6. Since Koseven is only PHP 7.0 Compatible this is not neccessary, but if you want i can revert

Reply:
Ok in that case this can be removed.